src/Modules/ConfigOptions.lua

```diff
@@ -1718,21 +1715,45 @@ Huge sets the radius to 11.
 	
 	-- Section: Custom mods
 	{ section = "Custom Modifiers", col = 1 },
-	{ var = "customMods", type = "text", label = "", apply = function(val, modList, enemyModList)
-		for line in val:gmatch("([^\n]*)\n?") do
-			local mods, extra = modLib.parseMod(line)
+	{ var = "customMods", type = "text", label = "",
+		apply = function(val, modList, enemyModList, build)
+			for line in val:gmatch("([^\n]*)\n?") do
+				local strippedLine = StripEscapes(line):gsub("^[%s?]+", ""):gsub("[%s?]+$", "")
+				local mods, extra = modLib.parseMod(strippedLine)
 
-			if mods then
-				local source = "Custom"
-				for i = 1, #mods do
-					local mod = mods[i]
+				if mods then
+					local source = "Custom"
+					for i = 1, #mods do
+						local mod = mods[i]
 
-					if mod then
-						mod = modLib.setSource(mod, source)
-						modList:AddMod(mod)
+						if mod then
+							mod = modLib.setSource(mod, source)
+							modList:AddMod(mod)
+						end
 					end
 				end
 			end
-		end
-	end },
+		end,
+		inactiveText = function(val)
+			local inactiveText = ""
+			for line in val:gmatch("([^\n]*)\n?") do
+				local strippedLine = StripEscapes(line):gsub("^[%s?]+", ""):gsub("[%s?]+$", "")
+				local mods, extra = modLib.parseMod(strippedLine)
+				inactiveText = inactiveText .. (mods and colorCodes.MAGIC or colorCodes.UNSUPPORTED).. (IsKeyDown("ALT") and strippedLine or line) .. "\n"
+			end
+			return inactiveText
+		end,
+		tooltip = function(modList)
+			if not launch.devModeAlt then
+				return
+			end
+
+			local out = ""
+			for _, mod in ipairs(modList) do
+				if mod.source == "Custom" then
+					out = (#out > 0 and out.."\n" or out) .. modLib.formatMod(mod) .. "|" .. mod.source
```

Comment:
Could do this if you don't initialize `out`.  This also means you don't have to have the `or tooltipText ~= ""` in the TooltipHost/ConfigTab logic above.

```suggestion
					out = (out and out.."\n" or "") .. modLib.formatMod(mod) .. "|" .. mod.source
```

Reply:
the tooltipText change comes from the previous commit, I think in lua empty string evaluates as true still and thats why I added those tooltipText checks.

Reply:
![image](https://user-images.githubusercontent.com/5115805/222538774-d7868342-45ec-4728-8f95-fc8833845aa1.png)

yea look

Reply:
Yeah, I'm saying if above you change line 1751 to `local out` instead.

Reply:
Oh, sure, can do that i guess. But I would need to change the other tooltips in prev commit too to do similar thing to remove the extra tooltip ~= "" check. But I dont think there is ever a reason to show empty tooltip anyway so I think that change makes sense anyway

Reply:
I hadn't looked at that other commit quite yet, but yeah I think stylistically we'll probably want to change those.  I can do that refactor possibly if I find the time.  Most of the code base doesn't use `if text ~= '' then`, it uses `if text then` instead (at least, off the top of my head)

Reply:
I will change that then its pretty easy when thats preferred

Reply:
Updated